chain/lf3/f3.go

```diff
@@ -184,20 +184,23 @@ func (fff *F3) GetLatestCert(ctx context.Context) (*certs.FinalityCertificate, e
 	return fff.inner.GetLatestCert(ctx)
 }
 
-func (fff *F3) GetManifest(ctx context.Context) *manifest.Manifest {
+func (fff *F3) GetManifest(ctx context.Context) (*manifest.Manifest, error) {
 	m := fff.inner.Manifest()
+	if m == nil {
+		return nil, xerrors.Errorf("no known network manifest")
```

Comment:
* `xerrors.New`?
* Do we want a specific type for this error?

Reply:
The sub API level errors don't matter really. But at API level I think Lotus should explicitly define its own error types, independent of its dependencies.

For this specific case the net effect is that the participant will retry anyway so up to you if you'd like to mae this an explicit error type or leave it as is. Either works i think.

Reply:
Raised in https://github.com/filecoin-project/go-f3/issues/795#issuecomment-2541104898 that I think it would be good to be able to type check against so we could transform them more reliably for Lotus. It would be pretty nice if we could do `errors.Is` checks on all of the errors in this list but I don't really want to achieve that by string comparisons of `Error()` values from what go-f3 passes on.